.changeset/upset-rice-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+Add `ToggleButtonGroupField` component and mark `FinalFormToggleButtonGroup` as deprecated.
```

Comment:
The second part should be a separate changeset for for cms-admin.

Reply:
^^ - I had this already, but i can change this.

Reply:
added an additional changset for this.